documentation/src/main/wiki/GettingStarted.md

```diff
@@ -260,3 +264,32 @@ This section configures the Services mock used by the example application. It co
 
 Please mind that this an example that depicts a valid setup of the Sample monolith application and it is not fit for use in production environments.
 To learn how to configure Knot.x for use in production, see the [Production](#configuration-1) section.
+
+## Deployment options
+To deploy verticle with advanced options use following properties:
+```json
+{
+    ... 
+  "mockService": {
+    "config": {
+      "mock.data.root": "mock/service",
+      "http.port": 3000
+    },
+    "worker" : false,
```

Comment:
Not exacly - having it as is it now - you don't need to do anything. vertx expects in json deployment options on current level, while config object is automaticlly read also. if you put deployment options into "deploymentOptions" then you'd need to handle it manually